akka-actor/src/main/scala/akka/event/EventBus.scala

```diff
@@ -180,6 +180,8 @@ trait SubchannelClassification { this: EventBus ⇒
    * Expensive call! Avoid calling directly from event bus subscribe / unsubscribe.
    */
   private[akka] def hasSubscriptions(subscriber: Subscriber): Boolean =
+    // FIXME binary incompatible, but I think it is safe to filter out this problem,
+    //       since it is only called from new functionality in EventStreamUnsubscriber
```

Comment:
the problem is that the method will not exist in user's classes, since this is a trait, until they compile, but I guess that is ok since they will not be using the new EventStreamUnsubscriber in that case anyway